orangecontrib/imageanalytics/hyper/stream.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+"""
+Temporary fix for hyper library until next version from 0.7.0 is out
+"""
```

Comment:
This file is under MIT license from hyper.

Reply:
Another thing to consider: development branch of hyper has a lot of new commits after the last version 0.7.0 (see https://github.com/Lukasa/hyper/compare/v0.7.0...development). You are fixing just one bug. Maybe it would be better to base our usage on development branch and not on 0.7.0?

In the mean time, as we're waiting for upstream to merge https://github.com/Lukasa/hyper/pull/356, we can use in requirements.txt your fixed branch
e.g.

pip install git+https://github.com/PrimozGodec/hyper@d8ed47aadbc1b66e8493726beeffcf0361ae8729

We switch to the upstream's version as soon the fix is merged. And nudge Lukasa to make another official version ;)

Reply:
If we plan to release a version of image-analytics add-on with this fix, I would rather have it base on a released version of the hyper package.

Otherwise, we are installing development version of hyper package into users environment, If we distribute over conda, we need to make sure hyper-devel builds are available, ...

Reply:
Ok if we are waiting for new release do I use my fixed branch anyway or leave as it is?

Reply:
@astaric I am not sure if we want to wait until next release. Hyper had a last release 0.7.0 more than one year ago. What means that releases are not so frequent. Is there any other solution since we are in hurry in margin this?

Reply:
I agree. I've asked to make e new release, but I don't know when it will be done. We can wait few days. Otherwise, we can just use a fixed commit ID in hyper and upgrade later? 

git+https://github.com/Lukasa/hyper@a8109c3aaf8e2aa7314f23bf46e20af2bc241cd7

Reply:
Feel free to use git+... in requirements and merge this PR.

But please do not create a (pypi) release that depends on development versions of packages.